posydon/utils/common_functions.py

```diff
@@ -44,7 +44,7 @@
                   "Shell_H_burning", "Central_He_depleted",
                   "Central_C_depletion"]
 RICHNESS_STATES = ["H-rich", "stripped_He"]
-COMPACT_OBJECTS = ["WD", "NS", "BH"]
+COMPACT_OBJECTS = ["WD", "NS", "BH","massless_remnant"]
```

Comment:
Let's be consistent throughout the code to have COMPACT_OBJECTS = ["WD", "NS", "BH"] and NOT_NORMAL_STARS = ["WD", "NS", "BH","massless_remnant"]